students/geonwoo/users/views.py

```diff
@@ -35,3 +35,20 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message":"KEY_ERROR"},status=400)
             
+class LoginView(View):
+    def post(self, request):
+        try:
+            data  = json.loads(request.body)
+           
+            if not User.objects.filter(email = data['email']).exists():
+                return JsonResponse({"message":"INVALID_USER"},status=401)
```

Comment:
```suggestion
            if not User.objects.filter(email = data['email'], (password = data['password').exists():
                return JsonResponse({"message":"INVALID_USER"},status=401)
```
아이디와 비밀번호가 동시에 맞는 유저를 찾아야 되기 때문에 filter 조건에 2가지 조건을 모두 넣으면 좋을 것 같습니다.